### ISSUE TITLE
invalid query name:

### ISSUE THREAD

**Issue by xx1196** [midir99/sqload#5]
i have this error
![image](https://user-images.githubusercontent.com/50302178/202233216-382cc745-2949-4969-b62d-cdca26b44b41.png)
 my sql folder is
![image](https://user-images.githubusercontent.com/50302178/202233350-7acab8c1-d28c-4841-87e6-0f406ab34ca9.png)
 i hope can help me plox

**Comment by midir99** [comment]
Hey!

I'm taking a look...

**Comment by midir99** [comment]
I tried whit the following and the library worked, could you please paste the code you are using here?

```sql
-- query: CreatePsychoCat
INSERT INTO Cat (name, color) VALUES ('Puca', 'Orange');
```

```go
package main

import (
	"embed"
	"fmt"

	"github.com/midir99/sqload"
)

//go:embed sql
var fsys embed.FS

var Q = sqload.MustLoadFromFS[struct {
	CreatePsychoCat string `query:"CreatePsychoCat"`
}](fsys)

func main() {
	fmt.Println(Q.CreatePsychoCat)
}
```

This is the regex used to validate the query name:
```go
var validQueryNamePattern = regexp.MustCompile(`^[a-zA-Z0-9_]+$`)`
```

**Comment by authapothc** [comment]
> please change this to LF mode 

![image](https://user-images.githubusercontent.com/5275512/205020870-4a712424-c019-4252-a26e-cda6c40e03a0.png)

**Comment by midir99** [comment]
Thanks for those comments, this will be fixed by the end of today.

**Comment by midir99** [comment]
Hey everyone, I just released version `v1.0.3`. To update just do:

```
$ go get -u github.com/midir99/sqload
```

This version adds support for files using `CRLF`. Please update when you have time. If you continue having issues please let me know. I'm going to keep this issue opened for few days just to make sure it's working for everyone.

Thank you all for your feedback, it's much appreciated ❤️.

**Comment by midir99** [comment]
I'm closing this issue since no new problems have reported and the merged PR adds support for files using `LF` or `CRLF`.
